src/core/internal/backtrace/handler.d

```diff
@@ -0,0 +1,116 @@
+/**
+ * Libunwind-based implementation of `TraceInfo`
+ *
+ * This module exposes an handler that uses libunwind to print stack traces.
+ * It is used when druntime is packaged with `DRuntime_Use_Libunwind` or when
+ * the user uses the following in `main`:
+ * ---
+ * import core.runtime;
+ * import core.internal.backtrace.handler;
+ * Runtime.traceHandler = &libunwindDefaultTraceHandler;
+ * ---
+ *
+ * Authors: Mathias 'Geod24' Lang
+ * Copyright: D Language Foundation - 2020
+ * See_Also: https://www.nongnu.org/libunwind/man/libunwind(3).html
```

Comment:
You're referring to LLVM libunwind in `core.internal.backtrace.libunwind`. Are they the same? Do they provide the same API.